docs/installing.rst

```diff
@@ -24,19 +24,18 @@ default conda-channels via::
 
     $ conda config --add channels conda-forge
 
-Then you can choose. Either you install straditize into your base environment
-via::
-
-    $ conda install -c chilipp straditize
-
-or you create a new virtual environment via::
+Then you can choose. We recommend to install straditize into it's own environment via::
```

Comment:
Should be `its` (without the apostrophe)

Reply:
Thanks! I resolved this by https://github.com/Chilipp/straditize/pull/6/commits/0e9f8af8db813470ded6c0af607b8fbf628a50b9